src/middlewared/debian/middlewared.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=TrueNAS Middleware
+DefaultDependencies=no
+
+Before=network-pre.target
+Wants=network-pre.target
+
+Wants=systemd-modules-load.service local-fs.target
+After=systemd-modules-load.service local-fs.target
+
+[Service]
+Type=notify
+ExecStart=/usr/bin/middlewared --log-handler=file
```

Comment:
Does it work if we add the truenas overlay directory?

Reply:
I think if I fix invalid imports, we'll be fine